File: website/src/components/Loan.js
import React, { Fragment } from 'react';

const loan = (props) => (
    <Fragment>
    <td>{props.id}</td>
    <td>{props.borrowerName}</td>
    <td>£{props.fundingAmount}</td>
    <td>£{props.repaymentAmount}</td>
    </Fragment>
)

export default loan;

File: WebAPI/lend-o-mat/Models/Loan.cs
namespace lend_o_mat.Models
{
    public class Loan
    {
        public long Id { get; set; }
        public string borrowerName { get; set; }
        public double repaymentAmount { get; set; }
        // the amount given to borrower (i.e. less than repayment amount)
        public double fundingAmount { get; set; }

        public Loan()
        {
        }
    }
}
